assets/xml/objects/object_link_boy.xml

```diff
@@ -0,0 +1,229 @@
+<Root>
+	<File Name="object_link_boy" Segment="6">
+		<!--Eyes-->
+        <Texture Name="gLinkAdultEyesOpenTex" OutName="link_adult_eyes_open" Format="ci8" TlutOffset="0x5C00" Width="64" Height="32" Offset="0x0"/>
+		<Texture Name="gLinkAdultEyesHalfTex" OutName="link_adult_eyes_half" Format="ci8" TlutOffset="0x5C00" Width="64" Height="32" Offset="0x800"/>
```

Comment:
I think you did yeah, was to be consistent, I can easily just remove it from both though if we want that, seems like a fine change since duplicate names won't matter because they are in a different directory anyways

Reply:
I'd say remove it then.  Since that is the standard.